system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -1204,8 +1209,12 @@ impl pallet_revive::Config for Runtime {
 	type CodeHashLockupDepositPercent = CodeHashLockupDepositPercent;
 	type ChainId = ConstU64<420_420_418>;
 	type NativeToEthRatio = ConstU32<1_000_000>; // 10^(18 - 12) Eth is 10^18, Native is 10^12.
-	type EthGasEncoder = ();
 	type FindAuthor = <Runtime as pallet_authorship::Config>::FindAuthor;
+	type AllowEVMBytecode = ConstBool<true>;
+	type FeeInfo = pallet_revive::evm::fees::Info<Address, Signature, EthExtraImpl>;
+	type MaxEthExtrinsicWeight = MaxEthExtrinsicWeight;
+	type DebugEnabled = ConstBool<false>;
```

Comment:
I think debug stuff can be feature gated behind some `debug` feature to never accidentally land in prod.

Reply:
This is not what features are meant for. We should only use them for additive things that are **harmless** when activated (i.e just add compilation time) when activated. Reasoning is that they can be implicitly activated through unification. Unlikely for a leave dependency but still.

We should rather rename the debug variable to `UnsafeDebug` to make clear that nobody should flip that switch.